src/frontend/scss/components/_header.scss

```diff
@@ -7,7 +7,7 @@
   background: r-theme-val(topbar, base-background);
 
   @include media-breakpoint-up($r-topbar-breakpoint) {
-    height: $r-topbar-height;
+    padding-bottom: 0;
```

Comment:
I stumbled upon this and quickly fixed it in this PR but it's kind of unrelated https://github.com/openfun/richie/pull/1706/commits/a2354f74e998ad45287222f6370a958317ef2289. *And* I'm actually not sure I didn't break some stuff, since the header is filled with templates blocks. I'm not sure that I can say to myself "this is okay" by just testing this with the demo site locally?

Reply:
Those blocks are usually user to change the header logo. We might need a `min-height` just to make sure it doesn't break. But we have to test this out.

Reply:
I think this property has been set to fix the size of the header but yeah in fact currently its content overflows.
But from my understanding, with your fix header height has no limit. According to the logo format (especially portrait) the header height could be very large. 

So I think we should set `max-height` to the logo then use `object-fit` and `object-position` to display it well.

Reply:
Ok oh well, I actually removed my commit from this branch. Since it's not really related to a11y, since I'm not sure what max height you'd want, and since I don't have any real other actually used logos from other instances of richie to test, I'd rather leave that out for now and spend time on other a11y stuff :)